enumerable.rb

```diff
@@ -0,0 +1,119 @@
+module Enumerable
+  def my_each
```

Comment:
block = proc { |num| num < (0 + 9 ) / 2 } }
Testing Range(5, 50).each(&block)  is not equal to Range(5,50).my_each(&block)